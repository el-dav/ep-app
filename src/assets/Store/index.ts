export { default as Store } from './Store.cmp';
