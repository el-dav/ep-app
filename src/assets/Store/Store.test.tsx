import React from 'react';
import renderer from 'react-test-renderer';

import { ThemeProvider } from 'theme';

import Store from './Store.cmp';

describe('Store', () => {
  it('renders the Component correctly', () => {
    const tree = renderer
      .create(
        <Store>
          <ThemeProvider />
        </Store>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
