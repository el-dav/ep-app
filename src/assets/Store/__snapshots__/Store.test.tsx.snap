// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Store renders the Component correctly 1`] = `null`;
